proto/options.proto

```diff
@@ -313,6 +313,9 @@ message Costing {
     float use_lit = 82;
     bool disable_hierarchy_pruning = 83;
     bool ignore_non_vehicular_restrictions = 84;
+    oneof has_hgv_no_access_penalty {
```

Comment:
has a default of `kMaxPenalty`

Reply:
would have be cool to have the default be 0, not use `oneof` and then just had 0 mean not allowed but i guess there is a tiny argument for you to be able to say its both allowed and the penalty is actually 0. anyway would have allowed us to not document the internal max penalty or been able to pick a smaller soft limit or something i dont i know haha